### ISSUE TITLE
Display the names of env variables, which fail to pass the validation

### ISSUE THREAD

**Issue by halfzebra** [Alexloof/centig#2]
Thanks for this library! 👍 
I'm having trouble finding what variables exactly are missing.

The current output is:

```
Error: 

Validation Error 
✘  Missing environment variable. Config name: host
✘  Missing environment variable. Config name: key
```

What I would have expected:

```
Centig Error:

✘  Missing environment variable "MY_HOST".

   At path: full.path.to.host

✘  Missing environment variable "MY_SECRET_KEY".

   At path: full.path.to.key

```

What do you think?

**Comment by Alexloof** [comment]
I see your point, a good idea. I will fix it 👍 @halfzebra

**Comment by Alexloof** [comment]
@halfzebra Fixed and released! Update your version to "1.0.12" to get the expected error outputs.

**Comment by halfzebra** [comment]
Thanks for the fix! 👍
